### PR TITLE
Notebook Examples for DP + GE: expect_column_values_vs_profile

### PR DESCRIPTION
**Pull request by micdavis**
Link to custom expectation used:
[expect_column_values_to_be_equal_to_or_less_than_profile_max](https://github.com/great-expectations/great_expectations/blob/develop/contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/expectations/expect_column_values_to_be_equal_to_or_less_than_profile_max.py)

### CODE REVIEW

**Comment by taylorfturner** [comment]
@micdavis add [this](https://github.com/capitalone/DataProfiler/pull/622#discussion_r961873357) to the README.md. Also conflict on `gitignore`

File: examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Now we will create a report on the first `individual_dataframe` which corresponds to the year 2020, then we will output the `min` metric from the `salary_in_usd` column as found in the report."
```
Comment (taylorfturner) [inline examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb:163]:
```suggestion
    "Now we will create a report on the first `individual_dataframe` which corresponds to the year **2020**."
```

Comment (taylorfturner) [reply examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb:163]: bold `2020` for for readability

File: examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
```
Comment (taylorfturner) [inline examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb:193]: Add a cell here for "Let's take a look at the output `min` metric from the `salary_in_usd` column as found in the report"

File: examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Running the Expectation\n",
+    "We build the validator by passing in the `individual_dataframe` corresponding to 2022. Then we will use the exception below to find that there are no values in the `salary_in_usd` column that exceed the `max` metric for the `age` column generated in `report`."
```
Comment (taylorfturner) [inline examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb:207]:
```suggestion
    "We build the validator by passing in the `individual_dataframe` corresponding to **2022**. Then we will use the exception below to find that there are no values in the `salary_in_usd` column that exceed the `max` metric for the `age` column generated in `report`."
```

just bolding

**Review by taylorfturner — COMMENTED** [review]
Comments... I think the rebase is off

File: examples/great_expecations/README.md [hunk 4]
```diff
@@ -10,6 +10,11 @@ Many others are contributed by the community of data practitioners that bring th
 See a full list of Data Profiler Expectations available through Great Expectations on the [package page](https://greatexpectations.io/packages/capitalone_dataprofiler_expectations). 
 If you are interested in learning more about expectations, here is a link to the [docs](https://greatexpectations.io/expectations).
 
+### About these Notebooks
+These notebook examples will utilize a dataset containing individual salary information across the world from the year 2020 until 2022.
+Each example will run through a different scenario that using the data and a custom expectation.
```
Comment (taylorfturner) [inline examples/great_expecations/README.md:15]:
```suggestion
Each example will run through a different scenario that uses the data and a custom Data Profiler expectation. 
```

File: examples/great_expecations/README.md [hunk 5]
```diff
@@ -10,6 +10,11 @@ Many others are contributed by the community of data practitioners that bring th
 See a full list of Data Profiler Expectations available through Great Expectations on the [package page](https://greatexpectations.io/packages/capitalone_dataprofiler_expectations). 
 If you are interested in learning more about expectations, here is a link to the [docs](https://greatexpectations.io/expectations).
 
+### About these Notebooks
+These notebook examples will utilize a dataset containing individual salary information across the world from the year 2020 until 2022.
+Each example will run through a different scenario that using the data and a custom expectation. 
+These scenarios should give a practical understanding of how expectations can be used in real world contexts as data quality checks.
```
Comment (taylorfturner) [inline examples/great_expecations/README.md:16]:
```suggestion
These scenarios provide a practical example of how expectations can be used in real world contexts for data quality checks.
```

File: examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "data_path = \"../../dataprofiler/tests/data/csv/ds_salaries.csv\"\n",
+    "df = pd.read_csv(data_path)\n",
```
Comment (taylorfturner) [inline examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb:90]:
```suggestion
    "df = dp.Data(data_path)\n",
```

**Review by taylorfturner — APPROVED** [review]

**Review by JGSweets — APPROVED** [review]

**Pull request merged**